dockernet/ts-tests/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
nit, wanna just move this to the main `.gitignore`

Reply:
a17d5c502